src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -310,7 +306,8 @@ export class AttachedAccountsTreeItem extends AzExtParentTreeItem {
                     isEmulator = (<IPersistedAccount>account).isEmulator;
                     label = isEmulator ? `${getExperienceFromApi(api).shortName} Emulator` : `${id} (${getExperienceFromApi(api).shortName})`;
                 }
-                const connectionString: string = nonNullValue(await keytar.getPassword(this._serviceName, id), 'connectionString');
+                // TODO: keytar: migration plan?
```

Comment:
TODO: do

Reply:
This is what the user is presented with:
![image](https://github.com/microsoft/vscode-cosmosdb/assets/36966225/de0a2831-2b85-444f-ba65-17f08995325f)

There are no attach actions on this tree item, though. Should add them.

Reply:
> There are no attach actions on this tree item, though. The user is forced to go through the command palette. Should add them.

If you merge with main, then you can use the new actions submenu. #2136